cli/tests/test_next_prev_commands.rs

```diff
@@ -567,6 +567,128 @@ fn test_next_editing() {
     "###);
 }
 
+#[test]
+fn test_prev_conflict() {
+    // Make the third commit our new parent.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let file_path = repo_path.join("content.txt");
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "first"]);
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "second"]);
+    std::fs::write(&file_path, "second").unwrap();
```

Comment:
It's confusing to write contents "second" in a commit called "third". Was this supposed to go into "second"? Or maybe it was supposed to write contents "third"

Reply:
this should now be "second" in the second commit.